web/src/components/CityContentToolbar.tsx

```diff
@@ -34,6 +51,20 @@ const CityContentToolbar = (props: CityContentToolbarProps) => {
   return (
     <Toolbar iconDirection={iconDirection} hideDivider={hideDivider}>
       {children}
+      <SocialMediaTooltip
+        active={shareOptionsVisible}
+        shareLink={window.location.href}
```

Comment:
Could we keep the naming URL internally?
```suggestion
        shareUrl={window.location.href}
```